tests/columns/test_map.py

```diff
@@ -22,7 +23,7 @@ def _sorted_dicts(self, text):
         return '\n'.join(items) + '\n'
 
     def test_simple(self):
-        with self.create_stream('a map(string, uint64)'):
+        with self.create_stream('a map(string, int)'):
```

Comment:
why change to int?

please remove the commented tests below if they are invalid.